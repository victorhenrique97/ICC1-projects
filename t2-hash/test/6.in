421
500
498
400
987
112
198
432
215
720
517
747
630
317
740
854
307
207
489
541
406
387
512
371
179
576
895
574
111
510
819
798
169
364
544
715
566
511
925
488
384
687
779
846
156
159
934
140
375
556
701
961
806
470
620
216
858
635
910
288
787
733
776
502
154
249
520
780
310
584
105
841
789
503
805
791
748
554
781
101
397
361
228
204
636
382
869
142
331
294
561
640
792
103
628
493
626
402
920
713
394
808
931
369
308
516
487
879
286
898
705
990
565
860
813
379
969
571
173
347
164
549
661
322
604
711
454
995
723
292
588
486
166
309
814
537
224
807
767
244
500
994
886
256
130
782
707
608
950
753
983
826
967
976
420
404
514
119
816
938
381
234
680
734
399
710
108
472
746
718
864
563
496
440
832
267
625
438
426
652
139
916
452
947
929
759
623
618
515
506
356
255
242
849
161
691
261
647
946
243
557
989
536
968
533
439
887
464
560
613
844
801
303
600
587
398
336
409
861
178
330
152
407
122
263
413
538
578
972
589
270
553
735
415
534
790
555
815
185
100
300
804
492
253
422
675
233
724
297
280
833
829
376
866
359
338
838
372
855
291
970
453
896
980
645
333
978
683
777
368
436
932
355
245
282
793
693
679
180
599
862
405
839
217
714
721
550
924
298
478
622
914
285
984
366
295
273
631
266
564
296
201
236
570
643
437
485
344
971
211
141
311
921
602
863
134
116
627
893
834
240
581
598
539
595
738
778
329
433
851
491
616
479
857
868
254
850
736
939
287
935
414
810
749
210
638
891
897
275
644
912
408
580
475
447
466
129
708
370
877
651
460
377
821
522
160
831
754
900
318
998
182
956
301
135
220
757
284
342
121
457
830
936
962
525
650
874
507
750
783
165
327
817
459
593
421
609
149
467
346
641
391
189
859
605
818
848
172
354
126
823
380
766
444
214
441
197
730
686
446
532
751
476
392
469
795
320
655
350
799
482
229
527
547
513
575
991
110
769
125
543
725
531
305
872
732
953
385
363
889
758
552
104
837
526
187
964
504
928
269
540
865
268
395
535
614
982
558
157
755
530
756
117
911
418
940
246
529
648
396
681
499
611
484
585
677
761
106
694
188
668
591
729
811
508
890
875
664
594
265
386
290
