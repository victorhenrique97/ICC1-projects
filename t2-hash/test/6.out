0: 421
1: 422
2: 844
3:
4: 846
5: 426
6: 848
7: 849
8: 850
9: 851
10:
11: 432
12: 854 433
13: 855
14:
15: 436 857
16: 858 437
17: 438 859
18: 860 439
19: 440 861
20: 862 441
21: 863
22: 864
23: 444 865
24: 866
25: 446
26: 868 447
27: 869
28:
29:
30: 872
31: 452
32: 453 874
33: 454 875
34:
35: 877
36: 457
37: 879
38: 459
39: 460
40:
41:
42:
43: 464
44: 886
45: 887 466
46: 467
47: 889
48: 469 890
49: 470 891
50:
51: 472 893
52:
53: 895
54: 896 475
55: 897 476
56: 898
57: 478
58: 479 900
59:
60:
61: 482
62:
63: 484
64: 485
65: 486
66: 487
67: 488
68: 489 910
69: 911
70: 491 912
71: 492
72: 493 914
73:
74: 916
75: 496
76:
77: 498
78: 920 499
79: 500 921
80:
81: 502
82: 503 924
83: 925 504
84:
85: 506
86: 507 928
87: 929 508
88:
89: 510 931
90: 511 932
91: 512
92: 934 513
93: 514 935
94: 515 936
95: 516
96: 517 938
97: 939
98: 940
99: 520
100: 100
101: 101 522
102:
103: 103
104: 946 525 104
105: 105 947 526
106: 527 106
107:
108: 950 108 529
109: 530
110: 110 531
111: 111 532 953
112: 112 533
113: 534
114: 956 535
115: 536
116: 537 116
117: 538 117
118: 539
119: 961 119 540
120: 541 962
121: 121
122: 122 543 964
123: 544
124:
125: 967 125
126: 968 126 547
127: 969
128: 549 970
129: 550 971 129
130: 130 972
131: 552
132: 553
133: 554
134: 976 555 134
135: 556 135
136: 557 978
137: 558
138: 980
139: 139 560
140: 140 561 982
141: 983 141
142: 142 563 984
143: 564
144: 565
145: 987 566
146:
147: 989
148: 990
149: 570 149 991
150: 571
151:
152: 994 152
153: 574 995
154: 154 575
155: 576
156: 156 998
157: 578 157
158:
159: 159 580
160: 581 160
161: 161
162:
163: 584
164: 164 585
165: 165
166: 166 587
167: 588
168: 589
169: 169
170: 591
171:
172: 593 172
173: 173 594
174: 595
175:
176:
177: 598
178: 178 599
179: 179 600
180: 180
181: 602
182: 182
183: 604
184: 605
185: 185
186:
187: 608 187
188: 609 188
189: 189
190: 611
191:
192: 613
193: 614
194:
195: 616
196:
197: 618 197
198: 198
199: 620
200:
201: 622 201
202: 623
203:
204: 204 625
205: 626
206: 627
207: 207 628
208:
209: 630
210: 631 210
211: 211
212:
213:
214: 635 214
215: 215 636
216: 216
217: 217 638
218:
219: 640
220: 220 641
221:
222: 643
223: 644
224: 224 645
225:
226: 647
227: 648
228: 228
229: 650 229
230: 651
231: 652
232:
233: 233
234: 234 655
235:
236: 236
237:
238:
239:
240: 661 240
241:
242: 242
243: 243 664
244: 244
245: 245
246: 246
247: 668
248:
249: 249
250:
251:
252:
253: 253
254: 675 254
255: 255
256: 256 677
257:
258: 679
259: 680
260: 681
261: 261
262: 683
263: 263
264:
265: 686 265
266: 687 266
267: 267
268: 268
269: 269
270: 691 270
271:
272: 693
273: 273 694
274:
275: 275
276:
277:
278:
279:
280: 701 280
281:
282: 282
283:
284: 705 284
285: 285
286: 286 707
287: 287 708
288: 288
289: 710
290: 711 290
291: 291
292: 713 292
293: 714
294: 715 294
295: 295
296: 296
297: 718 297
298: 298
299: 720
300: 300 721
301: 301
302: 723
303: 303 724
304: 725
305: 305
306:
307: 307
308: 308 729
309: 309 730
310: 310
311: 311 732
312: 733
313: 734
314: 735
315: 736
316:
317: 317 738
318: 318
319: 740
320: 320
321:
322: 322
323:
324:
325: 746
326: 747
327: 748 327
328: 749
329: 329 750
330: 330 751
331: 331
332: 753
333: 333 754
334: 755
335: 756
336: 336 757
337: 758
338: 759 338
339:
340: 761
341:
342: 342
343:
344: 344
345: 766
346: 767 346
347: 347
348: 769
349:
350: 350
351:
352:
353:
354: 354
355: 776 355
356: 356 777
357: 778
358: 779
359: 780 359
360: 781
361: 361 782
362: 783
363: 363
364: 364
365:
366: 787 366
367:
368: 789 368
369: 369 790
370: 791 370
371: 371 792
372: 372 793
373:
374: 795
375: 375
376: 376
377: 798 377
378: 799
379: 379
380: 801 380
381: 381
382: 382
383: 804
384: 384 805
385: 806 385
386: 807 386
387: 387 808
388:
389: 810
390: 811
391: 391
392: 813 392
393: 814
394: 394 815
395: 816 395
396: 817 396
397: 397 818
398: 819 398
399: 399
400: 400 821
401:
402: 402 823
403:
404: 404
405: 826 405
406: 406
407: 407
408: 829 408
409: 409 830
410: 831
411: 832
412: 833
413: 413 834
414: 414
415: 415
416: 837
417: 838
418: 839 418
419:
420: 841 420
